std/algorithm/setops.d

```diff
@@ -1117,8 +1118,9 @@ SetDifference!(less, R1, R2) setDifference(alias less = "a < b", R1, R2)
 }
 
 /**
-Lazily computes the intersection of two or more input ranges $(D
-ranges). The ranges are assumed to be sorted by `less`. The element
+Lazily computes the intersection of two or more
+$(REF_ALTTEXT input ranges, isInputRange, std,range,primitives)
+`ranges`. The ranges are assumed to be sorted by `less`. The element
```

Comment:
input ranges ranges

Reply:
That was in the original, and I assume it's referring to the parameter `ranges` of the function, hence the backticks.

Reply:
The parameter for the ctor is `input` which is a seperate ddoc. The parameter for this ddoc block is on line 1231 and is `Rs ranges`